src/router.js

```diff
@@ -46,6 +47,17 @@ var Router = {
       }
     });
 
+    try {
+      let rulesEngine = require('rules-engine/index.js');
+      // Issue a token without an associated user
```

Comment:
Again, rules engine code hard coded in the gateway means it isn't really a pluggable module, we'd have to add this code for every third party module.